csi/moac/test/watcher_test.js

```diff
@@ -230,7 +234,7 @@ module.exports = function () {
       const startTime = new Date();
       await watcher.create(apple);
       const delta = new Date() - startTime;
-      expect(delta).to.be.within(EVENT_TIMEOUT_MS, EVENT_TIMEOUT_MS + EYE_BLINK_MS);
+      expect(delta).to.be.within(EVENT_TIMEOUT_MS - TOLERATE_MS, EVENT_TIMEOUT_MS + EYE_BLINK_MS);
```

Comment:
Nit: could we assign this calculation to a "timeout" constant that way we only need to change it in one place?